packages/core/src/Services/socket-general.js

```diff
@@ -224,6 +224,7 @@ const BinarySocketGeneral = (() => {
                         'portfolio',
                         'proposal_open_contract',
                         'change_email',
+                        'phone_number_challenge',
```

Comment:
why do we add this here?

Reply:
We add this here to avoid users to get logs out when we get the error code of `InvalidToken`